### ISSUE TITLE
can't start new version

### ISSUE THREAD

**Issue by qrkourier** [openziti/desktop-edge-ui#106]
```bash
❯ apt show zitidesktopedge 
Package: zitidesktopedge
Version: 3.1.6
Status: install ok installed
Priority: optional
Section: default
Maintainer: Ziggy <ziggy@openziti.io>
Installed-Size: 314 MB
Depends: libgtk-3-0, libnotify4, libnss3, libxss1, libxtst6, xdg-utils, libatspi2.0-0, libuuid1, libsecret-1-0
Recommends: libappindicator3-1
Homepage: https://openziti.io
License: ISC
Vendor: Ziggy <ziggy@openziti.io>
Download-Size: unknown
APT-Manual-Installed: yes
APT-Sources: /var/lib/dpkg/status
Description: Ziti Desktop Edge Client


❯ zitidesktopedge 
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'find-remove'
Require stack:
- /opt/Ziti Desktop Edge (Preview)/resources/app.asar/app.js
- 
    at Module._resolveFilename (node:internal/modules/cjs/loader:940:15)
    at n._resolveFilename (node:electron/js2c/browser_init:249:1105)
    at Module._load (node:internal/modules/cjs/loader:785:27)
    at c._load (node:electron/js2c/asar_bundle:5:13343)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/opt/Ziti Desktop Edge (Preview)/resources/app.asar/app.js:12:20)
    at Module._compile (node:internal/modules/cjs/loader:1120:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1175:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
[1360570:0413/091802.962537:ERROR:vaapi_wrapper.cc(1299)] vaQuerySurfaceAttributes failed, VA error: invalid parameter
[1360570:0413/091802.962646:ERROR:vaapi_wrapper.cc(1246)] FillProfileInfo_Locked failed for va_profile VAProfileH264Main and entrypoint VAEntrypointVLD
[1360570:0413/091802.962696:ERROR:vaapi_wrapper.cc(1299)] vaQuerySurfaceAttributes failed, VA error: invalid parameter
[1360570:0413/091802.962738:ERROR:vaapi_wrapper.cc(1246)] FillProfileInfo_Locked failed for va_profile VAProfileH264High and entrypoint VAEntrypointVLD
```

**Comment by qrkourier** [comment]
I can run from branch `main` like this without the same error.

```bash
(cd ./src && npm install --no-save && npm start)
```

**Comment by JeremyTellier** [comment]
Shoot, I will make a release today, newbie mistake

**Comment by JeremyTellier** [comment]
Fixed in 3.1.7
